shared/presentation/src/commonMain/kotlin/network/bisq/mobile/presentation/ui/components/atoms/SegmentButton.kt

```diff
@@ -0,0 +1,30 @@
+package network.bisq.mobile.presentation.ui.components.atoms
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.setValue
+import androidx.compose.runtime.mutableIntStateOf
+import androidx.compose.runtime.remember
+
+// TODO: SingleChoiceSegmentedButtonRow: Not available in KMP?
```

Comment:
could be, not all the compose might be fully implemented. Would this help?

```
@Composable
fun SingleChoiceSegmentedButtonRow(
    options: List<String>,
    selectedOption: String,
    onOptionSelected: (String) -> Unit
) {
    Row {
        options.forEach { option ->
            TextButton(
                onClick = { onOptionSelected(option) },
                colors = if (option == selectedOption) {
                    ButtonDefaults.textButtonColors(
                        backgroundColor = MaterialTheme.colors.primary
                    )
                } else {
                    ButtonDefaults.textButtonColors(
                        backgroundColor = Color.Transparent
                    )
                }
            ) {
                Text(option)
            }
        }
    }
}
    
```

Reply:
Yeah. I will do this.
I didn't like doing the dropdown in Market filter.

Since it has only
 * 2 option for filter and
 * 3 options for sort
 
 It would look good, if displayed as SegmentedButtons.